packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v1-4-x/run-transaction-endpoint-v1.test.ts

```diff
@@ -173,6 +174,23 @@ test("runs tx on a Fabric v1.4.8 ledger", async (t: Test) => {
     t.ok(car277.Record.owner, `Car object has "Record"."owner" property OK`);
     t.equal(car277.Record.owner, carOwner, `Car has expected owner OK`);
   }
-
+  await new Promise((resolve) => setTimeout(resolve, 200000));
+  {
+    const promExporter = plugin.getPrometheusExporter();
```

Comment:
@jagpreetsinghsasan How will this work in a browser environment where you don't have access to the `PluginLedgerConnectorFabric` instance?

Reply:
I have fixed in the new commit.